test/input.txt

```diff
@@ -106,6 +107,22 @@
   # find multiple with some keywords
   find Charlie Betsy
 
+##########################################################
+# test favourite person command
+##########################################################
+
+# last active view: [1] betsy [2] charlie
+
+  # should catch invalid args format
+  favourite
+  favourite should be only one number
+
+  # should catch invalid index
+  favourite -1
+  favourite 0
+  favourite 3
+  favourite 2
+
```

Comment:
Good that I/O testing is updated
(Though because you didn't test for PersonNotFound, I can safely assumed that my thesis about "inheriting" is correct in my previous comment.)